lib/ui/mockData/MockDogTagViewModel.dart

```diff
@@ -1,5 +1,6 @@
 import 'package:farmsmart_flutter/ui/common/Dogtag.dart';
 import 'package:farmsmart_flutter/ui/common/MockString.dart';
+import 'package:flutter/material.dart';
```

Comment:
This is not part of this PR

Reply:
![PRAssistant](https://user-images.githubusercontent.com/23307893/61110516-b83c2380-a487-11e9-8517-ae3b7ceb093e.png)